apps/etterlatte-vedtaksvurdering/.nais/dev.yaml

```diff
@@ -111,17 +111,8 @@ spec:
         - application: etterlatte-beregning
         - application: etterlatte-testdata-behandler
         - application: etterlatte-samordning-vedtak
-          permissions:
-            roles:
-              - samordning-read
         - application: etterlatte-api    # tar over for samordning-vedtak
-          permissions:
-            roles:
-              - samordning-read
         - application: etterlatte-vedtaksvurdering-kafka
-          permissions:
-            roles:
-              - samordning-write
         - application: etterlatte-utbetaling
         - application: etterlatte-statistikk # skal fjernes etter at vedtaksdata i statistikk er oppdatert
```

Comment:
Ikkje relevant for PR-en, men tida for å fjerne denne er kanskje inne?

Reply:
Kanskje, jobben har ikke kjørt siden 21 juni ser jeg.

Reply:
Fant ingen god info på om det er tidsbegrenset her https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/5199 eller i jiraen.

Reply:
https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/5394